playbooks/ansible-test-integration-base/pre.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: controller
+  tasks:
+    - name: Install binary dependencies from ansible/ansible-zuul-jobs
+      include_role:
+        name: bindep
+      vars:
+        bindep_dir: "{{ zuul.projects['github.com/ansible/ansible-zuul-jobs'].src_dir }}"
```

Comment:
I actually don't think we need this, it was only used in network-integrations to bootstrap appliance dependencies